python/xir/program.py

```diff
@@ -563,6 +578,65 @@ def serialize(self, minimize: bool = False) -> str:
             return strip(res_script)
         return res_script
 
+    @staticmethod
+    def merge(*programs) -> "XIRProgram":
+        """Merges one or more XIR programs into a new XIR program.
+
+        The merged XIR program is formed by concatenating the given XIR programs
+        in the order they are passed to the function. Warnings may be issued for
+        duplicate declarations, gates, includes, operators, and options.
+
+        Args:
+            programs (XIRProgram): XIR programs to merge
+
+        Returns:
+            XIRProgram: the merged XIR program
+
+        Raises:
+            ValueError: if no XIR programs are provided or if at least two XIR
+                programs have different versions or float settings
+        """
+        if len(programs) == 0:
+            raise ValueError("Merging requires at least one XIR program.")
```

Comment:
So, basically, merging a single program would be the same as copying it. 👀

Reply:
Yes although I would say this is more of a happy accident than anything else.